src/main/java/shop/entity/Order.java

```diff
@@ -37,6 +36,10 @@ public void setStatus(String a) {
         return productList;
     }
 
+    public void addList(Product product){
```

Comment:
change to public void setProductList(List<Product> productList){
this.productList = productList;
}